src/ConfigurationException/InvalidFixerConfigurationException.php

```diff
@@ -21,7 +21,7 @@
  *
  * @internal
  */
-class InvalidFixerConfigurationException extends InvalidConfigurationException
+final class InvalidFixerConfigurationException extends InvalidConfigurationException
```

Comment:
won't work actually as `InvalidForEnvFixerConfigurationException` is extending this class.
initial `@final` annotation mislead me